lib/oli_web/live/delivery/student/content_live.ex

```diff
@@ -3,23 +3,391 @@ defmodule OliWeb.Delivery.Student.ContentLive do
+          }
+        />
+        <.module_card
+          :for={{module, module_index} <- Enum.with_index(@unit.children, 1)}
+          module={module}
+          module_index={module_index}
+          unit_uuid={@unit.uuid}
+          unit_numbering_index={@unit.numbering.index}
+          bg_image_url={module.revision.poster_image}
+          selected={if @selected_module, do: module.uuid == @selected_module.uuid, else: false}
+        />
+      </div>
+    </div>
+    <div :if={@selected_unit} class="flex py-[24px] px-[50px] gap-x-4 lg:gap-x-12">
+      <div class="w-1/2 flex flex-col px-6">
+        <div class={[
+          "intro-content",
+          maybe_additional_margin_top(@selected_module.revision.intro_content["children"])
+        ]}>
+          <%= Phoenix.HTML.raw(
+            Oli.Rendering.Content.render(
+              %Oli.Rendering.Context{},
+              @selected_module.revision.intro_content["children"],
+              Oli.Rendering.Content.Html
+            )
+          ) %>
+        </div>
+        <button class="btn btn-primary mr-auto mt-[42px]">Let's discuss?</button>
+      </div>
+      <div class="mt-[62px] w-1/2">
+        <.index module={@selected_module} module_index={@selected_module_index} />
+        <p class="py-2 text-[14px] leading-[30px] font-normal"></p>
+      </div>
+    </div>
+    """
+  end
+
+  attr :module, :map
+  attr :module_index, :integer
+
+  def index(assigns) do
+    # TODO: render real check if student has visited the page
+    ~H"""
+    <div
+      :for={{page, page_index} <- Enum.with_index(@module.children, 1)}
+      class="flex gap-[14px] h-[42px] w-full"
+    >
+      <div class="flex justify-center items-center gap-[10px] h-6 w-6">
+        <svg
+          :if={:rand.uniform() > 0.5}
+          xmlns="http://www.w3.org/2000/svg"
+          height="1.25em"
+          viewBox="0 0 448 512"
+        >
+          <path
+            fill="#1E9531"
+            d="M438.6 105.4c12.5 12.5 12.5 32.8 0 45.3l-256 256c-12.5 12.5-32.8 12.5-45.3 0l-128-128c-12.5-12.5-12.5-32.8 0-45.3s32.8-12.5 45.3 0L160 338.7 393.4 105.4c12.5-12.5 32.8-12.5 45.3 0z"
+          />
+        </svg>
+      </div>
+      <div
+        phx-click="navigate_to_resource"
+        phx-value-slug={page.revision.slug}
+        class="flex items-center gap-3 w-full border-b-2 border-gray-600 cursor-pointer hover:bg-gray-200/70 px-2"
+      >
+        <span class="text-[16px] leading-[22px] font-normal truncate">
+          <%= "#{@module_index}.#{page_index} #{page.revision.title}" %>
+        </span>
+        <div class="flex items-center gap-[6px] ml-auto">
+          <svg xmlns="http://www.w3.org/2000/svg" height="1.25em" viewBox="0 0 512 512">
+            <path d="M464 256A208 208 0 1 1 48 256a208 208 0 1 1 416 0zM0 256a256 256 0 1 0 512 0A256 256 0 1 0 0 256zM232 120V256c0 8 4 15.5 10.7 20l96 64c11 7.4 25.9 4.4 33.3-6.7s4.4-25.9-6.7-33.3L280 243.2V120c0-13.3-10.7-24-24-24s-24 10.7-24 24z" />
+          </svg>
```

Comment:
I know this is a temporary solution, but if we don't get the assets soon, we could define this svgs in separate files and import them to improve readability.

Something like
```html
<img src="progress_bar.svg" />
```

Reply:
yes, makes sense. I hope to get the assets soon and, if not, I will define those in separate files.